### ISSUE TITLE
Expand WG Charter to allow additional documents and features per staging process

### ISSUE THREAD

**Issue by evanp** [swicg/potential-charters#37]
The staging process presumes that new ideas will move from stages 0-2 and then be considered for recommendation status.

If this is going to be the case, we should give the WG sufficient leeway to actually create new recommendations or add features to our existing ones.

**Comment by gobengo** [comment]
Personally I would like to see any WG focus as much of its initial efforts as possible on reaching consensus on errata in ActivityPub and updating the TR accordingly to improve the implementability of the spec and unblock new implementors as well as any projects like reference implementations and conformance test suites that require a stable spec with errata incorporated.

I think the staging process is a good idea to include in a CG Charter and decision policy. WGs by convention can collaborate with CGs and incubate work in the CG through the staging process (i.e. incubation process). I don't think we need mention of the staging process in any WG charter. e.g. the FedCM WG charter doesn't seem to mention the related staging process. https://www.w3.org/2024/03/wg-fedid-charter.html Some WG work items were adopted from the CG after going through the process, but I think that can in general be a charter update to deliverables, and the WG charter need not mention the CG incubation process at all (as with FedCM).

**Comment by evanp** [comment]
The WG charter says no new features.

The staging process probably would stop around Stage 2 (Report) if the WG can't add new features or documents.

If we want the WG to add new docs and features, we should give it the power to do so.

**Comment by evanp** [comment]
[The charter](https://swicg.github.io/potential-charters/ap-maintenance-wg-charter.html#out-of-scope) says:

> Changes that add new functionality ([class 4](https://www.w3.org/policies/process/#class-4)) are out of scope.

The referenced policy says:

> Changes that add new functionality, such as new elements, new APIs, new rules, etc.

The stage process, on the other hand, [says](https://swicg.github.io/potential-charters/stage-process#stage-2-formalization-community-group):

> What’s asked of the Working Group?
> - Working Group consensus to adopt the proposal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
> - Working Group identification of the list of (seemingly resolvable) issues that have to be addressed before [Stage 3](https://swicg.github.io/potential-charters/stage-process#stage-3-implementation-and-normative-draft-working-group).
> - An [Editor’s Draft](https://www.w3.org/policies/process/#editors-draft) can be used to get PRs merged between [Working Draft](https://www.w3.org/policies/process/#RecsWD) revisions.

That sounds like we expect new features to be in scope for the WG. If so, I think we should adjust the scope of the WG. That doesn't require any mention of the staging process in the actual charter.

**Comment by evanp** [comment]
I think we covered this in today's conversation, and we're going to include class 4 as in-scope?
